src/bootstrap/flags.rs

```diff
@@ -483,7 +483,12 @@ Arguments:
         }
 
         let cmd = match subcommand.as_str() {
-            "build" | "b" => Subcommand::Build { paths },
+            "build" | "b" => {
+                if paths.is_empty() {
+                    paths.push(PathBuf::from("library/std"));
```

Comment:
This feels very odd. This seems no different than just removing DEFAULT = true from all of the build-called steps except std; why is that not the approach taken here?